src/libcmd/installables.cc

```diff
@@ -858,7 +858,7 @@ std::vector<FlakeRef> RawInstallablesCommand::getFlakeRefsForCompletion()
     applyDefaultInstallables(rawInstallables);
     std::vector<FlakeRef> res;
     res.reserve(rawInstallables.size());
-    for (auto i : rawInstallables)
+    for (const auto& i : rawInstallables)
```

Comment:
Style nitpick:
```suggestion
    for (const auto & i : rawInstallables)
```
and everywhere else.

Reply:
Does our clang-format configuration follow the same rules?

Reply:
Yes (`PointerAlignment: Middle`).

Reply:
Thanks, fixed the style. It's still pretty hard to format diffs, since clangd range formatting sometimes touches more code than necessary.